### PR TITLE
Fixes #421 Organize Fate Operations

### LINKED ISSUE #421: Organize Fate Operations
The RepOs of every fate operation are ALL in the org.apache.accumulo.master.tableOps package.  It is annoying to track down which RepO goes with which operation.  Creating a package in o.a.a.master.tableOps for each operation would organize them nicely.  This would have to be done in 2.0 since it will break serialization.

### PR DESCRIPTION
**Pull request by cjmctague**
- clean up tableOps directory to be organized by Fate operation.

### CODE REVIEW

**Comment by milleruntime** [comment]
@cjmctague At first glance, this looks great!  I am going to checkout your branch and take a closer look today.

File: server/master/src/main/java/org/apache/accumulo/master/tableOps/create/CreateTable.java [hunk 1]
```diff
@@ -62,18 +65,19 @@ public long isReady(long tid, Master environment) throws Exception {
 
     // assuming only the master process is creating tables
 
-    Utils.idLock.lock();
+    Utils.getIdLock().lock();
     try {
-      tableInfo.tableId = Utils.getNextId(tableInfo.tableName, master.getContext(), Table.ID::of);
+      tableInfo
+          .setTableId(Utils.getNextId(tableInfo.getTableName(), master.getContext(), Table.ID::of));
```
Comment (milleruntime) [inline server/master/src/main/java/org/apache/accumulo/master/tableOps/create/CreateTable.java]: This was probably autoformat'ed but you could put a line break somewhere else.

Comment (cjmctague) [reply server/master/src/main/java/org/apache/accumulo/master/tableOps/create/CreateTable.java]: Is there a way to turn off the auto formatter for that line cause I believe it's just gonna get reverted back the next time it runs.

Comment (milleruntime) [reply server/master/src/main/java/org/apache/accumulo/master/tableOps/create/CreateTable.java]: It's OK the the way it is, I'm not sure messing with the formatter looks much better anyway.  But to turn the formatter off and then back on requires these lines:
// @formatter:off
...
// @formatter:on

Comment (ctubbsii) [reply server/master/src/main/java/org/apache/accumulo/master/tableOps/create/CreateTable.java]: I don't think it's worth messing with the formatter enabling/disabling here. Abuse of that can lead to some really ugly code over time. It looks fine to me as is.

Comment (ctubbsii) [reply server/master/src/main/java/org/apache/accumulo/master/tableOps/create/CreateTable.java]: Also, I'd have to double check, but I think master *is* a context, so I'm not sure you need to pass in `master.getContext()` when you can just pass in `master`. If I'm right, that could shorten the line enough to make this look prettier.

Comment (milleruntime) [reply server/master/src/main/java/org/apache/accumulo/master/tableOps/create/CreateTable.java]: If that doesn't work you could shorten "tableInfo" to tInfo or just info

Comment (milleruntime) [reply server/master/src/main/java/org/apache/accumulo/master/tableOps/create/CreateTable.java]: Or create a local variable above:
<code>String name = tableInfo.getTableName();</code>
Which will save you a bunch of characters on that line.

Comment (cjmctague) [reply server/master/src/main/java/org/apache/accumulo/master/tableOps/create/CreateTable.java]: `master` wasn't a context, unfortunately. I just used a local variable to shorten the line so it won't auto format later down the road.

**Comment by milleruntime** [comment]
OK looks good, I will merge.

**Comment by milleruntime** [comment]
Actually, @cjmctague do you mind resolving the conflict with CleanUp.java?  Looks like recent changes caused a conflict.

**Comment by milleruntime** [comment]
I also fired off a Jenkins build which will run all the tests to make sure everything is cool. https://jenkins.revelc.net/job/Accumulo-PR/84/

**Comment by cjmctague** [comment]
@milleruntime Rebased and squashed into one commit.

**Comment by milleruntime** [comment]
The PR jenkins build looks good.  I reran the Travis Hadoop 2 build but it is still dying halfway through.  I don't believe this is related to these changes so I will merge.

**Pull request merged**